packages/zeek/data_stream/known_certs/elasticsearch/ingest_pipeline/default.yml

```
@@ -104,6 +104,7 @@ processors:
         - 'CN=%{CN:tls.server.x509.subject.common_name}'
       pattern_definitions:
         CN: '[^,]+'
+      if: ctx?.tls?.server?.x509?.subject?.distinguished_name != null && ctx.tls.server.x509.subject.distinguished_name.contains('CN=')
```

```suggestion
      if: ctx.tls?.server?.x509?.subject?.distinguished_name != null && ctx.tls.server.x509.subject.distinguished_name.contains('CN=')
```

`ctx` is never null.